--- binutils-2.31/ld/configure.tgt	2018-07-09 07:44:53.000000000 +0000
+++ binutils-2.31/ld/configure.tgt.new	2018-11-11 18:06:56.650000000 +0000
@@ -775,15 +775,15 @@
 
 esac
 
-NATIVE_LIB_DIRS='/usr/local/lib /lib /usr/lib'
+NATIVE_LIB_DIRS='REPLACE/lib'
 case "${target}" in
 
 *-*-dragonfly*)
-  NATIVE_LIB_DIRS='/lib /usr/lib /usr/pkg/lib /usr/local/lib'
+  NATIVE_LIB_DIRS='REPLACE/lib'
   ;;
 
 *-*-freebsd*)
-  NATIVE_LIB_DIRS='/lib /usr/lib /usr/local/lib'
+  NATIVE_LIB_DIRS='REPLACE/lib'
   ;;
 
 hppa*64*-*-hpux11*)
