--- gcc-8.2.0/gcc/config.gcc	2018-12-14 13:39:05.250850356 -0600
+++ gcc-8.2.0/gcc/config.gcc.new	2018-12-14 13:41:11.094850126 -0600
@@ -227,7 +227,7 @@
 default_gnu_indirect_function=no
 target_gtfiles=
 need_64bit_isa=
-native_system_header_dir=/usr/include
+native_system_header_dir=REPLACE/include
 target_type_format_char='@'
 
 # Don't carry these over build->host->target.  Please.
@@ -759,7 +759,7 @@
     *-*-linux* | frv-*-*linux* | *-*-kfreebsd*-gnu | *-*-kopensolaris*-gnu)
       :;;
     *-*-gnu*)
-      native_system_header_dir=/include
+      native_system_header_dir=REPLACE/include
       ;;
   esac
   # Linux C libraries selection switch: glibc / uclibc / bionic.
@@ -4450,8 +4450,8 @@
 			 echo "#undef  LINK_OS_EXTRA_SPEC64"
 			 echo "#define LINK_OS_EXTRA_SPEC64" \
 			      "\"%(link_os_new_dtags)" \
-			      "-rpath $prefix/lib64 -rpath $at/lib64" \
-			      "-L $prefix/lib64 -L $at/lib64\""
+			      "-rpath $prefix/lib -rpath $at/lib" \
+			      "-L $prefix/lib -L $at/lib\""
 			 echo
 			 echo "#undef  LINK_OS_NEW_DTAGS_SPEC"
 			 echo "#define LINK_OS_NEW_DTAGS_SPEC" \
