--- gcc-8.2.0/gcc/gcc.c	2018-02-09 06:44:06.000000000 +0000
+++ gcc-8.2.0/gcc/gcc.new	2018-11-11 15:46:59.667000000 +0000
@@ -1462,10 +1462,10 @@
 /* Default prefixes to attach to command names.  */
 
 #ifndef STANDARD_STARTFILE_PREFIX_1
-#define STANDARD_STARTFILE_PREFIX_1 "/lib/"
+#define STANDARD_STARTFILE_PREFIX_1 "REPLACE/lib/"
 #endif
 #ifndef STANDARD_STARTFILE_PREFIX_2
-#define STANDARD_STARTFILE_PREFIX_2 "/usr/lib/"
+#define STANDARD_STARTFILE_PREFIX_2 "REPLACE/lib/"
 #endif
 
 #ifdef CROSS_DIRECTORY_STRUCTURE  /* Don't use these prefixes for a cross compiler.  */
