--- glibc-2.28/sysdeps/unix/sysv/linux/x86_64/ldconfig.h	2018-08-01 00:10:47.000000000 -0500
+++ glibc-2.28/ldconfig.h.new	2018-12-03 11:13:58.549286339 -0600
@@ -18,9 +18,9 @@
 #include <sysdeps/generic/ldconfig.h>
 
 #define SYSDEP_KNOWN_INTERPRETER_NAMES \
-  { "/lib/ld-linux.so.2", FLAG_ELF_LIBC6 }, \
-  { "/libx32/ld-linux-x32.so.2", FLAG_ELF_LIBC6 }, \
-  { "/lib64/ld-linux-x86-64.so.2", FLAG_ELF_LIBC6 },
+  { "REPLACE/lib/ld-linux.so.2", FLAG_ELF_LIBC6 }, \
+  { "REPLACE/lib/ld-linux-x32.so.2", FLAG_ELF_LIBC6 }, \
+  { "REPLACE/lib/ld-linux-x86-64.so.2", FLAG_ELF_LIBC6 },
 #define SYSDEP_KNOWN_LIBRARY_NAMES \
   { "libc.so.6", FLAG_ELF_LIBC6 },	\
   { "libm.so.6", FLAG_ELF_LIBC6 },
